applications/bci_visualization/operators/voxel_stream_to_volume/voxel_stream_to_volume.py

```
@@ -0,0 +1,341 @@
+        self.mask_volume_gpu = None
+        self.mask_affine = None
+        self.mask_shape = None
+        
+        # Running statistics for adaptive normalization (initialized from first frame)
+        self.global_min = None
+        self.global_max = None
+        self.frame_count = 0
+
+    def start(self):
+        if not self.mask_nifti_path:
+            raise ValueError("VoxelStreamToVolume: No mask NIfTI path provided")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)